### ISSUE TITLE
LL can't start the Edge browser locally

### ISSUE THREAD

**Issue by mark0203** [spriteCloud/lapis-lazuli#77]
Both Watir and Selenium can start the Edge browser locally by doing:
```
#watir
require 'watir-webdriver' 
browser1 = Watir::Browser.new 'edge'

#selenium
require 'selenium-webdriver'
browser2 = Selenium::WebDriver.for(:edge)
```

Currently the only way for me to connect with LL to Edge is via selenium-grid
```
browser :remote, {
  :url => "http://[url to selenium-grid hub]/wd/hub",
  :caps => {
    "browserName" => "MicrosoftEdge",
  }
}
```	

It would be nice to also be able to start Edge locally without the need for a selenium-grid node.

**Comment by mauricewijnia** [comment]
Pull request #83 fixes this issue

**Comment by barzilay** [comment]
Fixed in [LL 2.0.1](https://rubygems.org/gems/lapis_lazuli/versions/2.0.1)
